real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/nodes/localizers/LocalizerUtilTest.java

```diff
@@ -0,0 +1,70 @@
+package com.roboclub.robobuggy.nodes.localizers;
+
+import com.roboclub.robobuggy.ui.LocTuple;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static java.lang.Math.PI;
+import static java.lang.Math.sqrt;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests the Localizer util functions
+ */
+public class LocalizerUtilTest {
+
+    private final double latPittsburgh = 40.440310;
+    private final double lonPittsburgh = -79.9471537;
+    private final double calcedDistOneDegMeters = 84721.0;
+
+    // the distance in meters for 1˚ up and 1˚ right
+    private final double diagDistMeters = 139505.0;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2337031408/issues/source?bid=2938665&fileBranchId=3199623#l22)